services/grafana/client_test.go

```diff
@@ -163,7 +163,7 @@ func TestClient(t *testing.T) {
 			require.NoError(t, err)
 			for _, a := range annotations {
 				if a.Text == "No tags" {
-					assert.Equal(t, []string{"pmm_annotation"}, a.Tags)
+					assert.Equal(t, []string{}, a.Tags)
```

Comment:
maybe check for zero length?

Reply:
Sure I check check len for Tags. Will change it.

Reply:
check should be changed in assert, not in if. :)

Reply:
Sorry, I read it wrong. Fixed.